com.substance3d.launcher.desktop

```
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Name=Substance Launcher
+Comment=Product aquisition for Allegorithmic Software
+Exec=substance
+Terminal=false
+Type=Application
+Icon=com.substance3d.launcher
+StartupWMClass=Substance Launcher
+Categories=GNOME;GTK;Graphics;
```

It's not a GNOME nor a GTK app.
```suggestion
Categories=Graphics;
```

---

This is what the official .desktop file has, but yes, it makes sense to remove them.